<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureDialog</class>
 <widget class="QDialog" name="ConfigureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure RIVET</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>561</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="colorsTab">
    <attribute name="title">
     <string>Colors</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>511</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Select colors for the visualizations:</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>401</width>
       <height>250</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="2">
       <widget class="QPushButton" name="xi0colorButton">
        <property name="toolTip">
         <string>click to change color</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="xi1colorLabel">
        <property name="text">
         <string>ξ₁ points</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="xi0colorLabel">
        <property name="text">
         <string>ξ₀ points</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="persistenceHighlightColorLabel">
        <property name="text">
         <string>Persistence diagram highlight</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="lineColorLabel">
        <property name="text">
         <string>Line selection</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="persistenceColorLabel">
        <property name="text">
         <string>Persistence diagram</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="lineHighlightColorLabel">
        <property name="text">
         <string>Line selection highlight</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="xi1colorButton">
        <property name="toolTip">
         <string>click to change color</string>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="persistenceColorButton">
        <property name="toolTip">
         <string>click to change color</string>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="persistenceHighlightColorButton">
        <property name="toolTip">
         <string>click to change color</string>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="lineColorButton">
        <property name="toolTip">
         <string>click to change color</string>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="lineHighlightColorButton">
        <property name="toolTip">
         <string>click to change color</string>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="xi0spinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>0 is fully transparent, 255 is fully opaque</string>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QSpinBox" name="persistenceHighlightSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>0 is fully transparent, 255 is fully opaque</string>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QSpinBox" name="lineSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>0 is fully transparent, 255 is fully opaque</string>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSpinBox" name="persistenceSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>0 is fully transparent, 255 is fully opaque</string>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="xi1spinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>0 is fully transparent, 255 is fully opaque</string>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QSpinBox" name="lineHighlightSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>0 is fully transparent, 255 is fully opaque</string>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="colorLabel">
        <property name="text">
         <string>Color</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="colorLabel_2">
        <property name="text">
         <string>Transparency</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="xi2colorLabel">
        <property name="text">
         <string>ξ₂ points</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="xi2colorButton">
        <property name="toolTip">
         <string>click to change color</string>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSpinBox" name="xi2spinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>0 is fully transparent, 255 is fully opaque</string>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="defaultColorsButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>190</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Restore Default Colors</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sizesTab">
    <attribute name="title">
     <string>Sizes</string>
    </attribute>
    <widget class="QLabel" name="dotSizeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Radius for unit dots, in pixels:</string>
     </property>
    </widget>
    <widget class="QLabel" name="bettiDotSizeLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>45</y>
       <width>331</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Support points of multigraded Betti numbers</string>
     </property>
    </widget>
    <widget class="QLabel" name="persistenceDotSizeLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>75</y>
       <width>291</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Persistence diagram dots</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="bettiDotSpinBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="persistenceDotSpinBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QCheckBox" name="AutoDotSizeCheckBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Use automatic dot sizes</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="labelsTab">
    <attribute name="title">
     <string>Labels</string>
    </attribute>
    <widget class="QLabel" name="axisLabelLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Line Selection Window Axis Labels</string>
     </property>
    </widget>
    <widget class="QLabel" name="xaxisLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>x-axis:</string>
     </property>
    </widget>
    <widget class="QLabel" name="yaxisLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>y-axis:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="xaxisText">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>35</y>
       <width>251</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="yaxisText">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>65</y>
       <width>251</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>380</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>380</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>xi0colorButton</tabstop>
  <tabstop>xi0spinBox</tabstop>
  <tabstop>xi1colorButton</tabstop>
  <tabstop>xi1spinBox</tabstop>
  <tabstop>xi2colorButton</tabstop>
  <tabstop>xi2spinBox</tabstop>
  <tabstop>persistenceColorButton</tabstop>
  <tabstop>persistenceSpinBox</tabstop>
  <tabstop>persistenceHighlightColorButton</tabstop>
  <tabstop>persistenceHighlightSpinBox</tabstop>
  <tabstop>lineColorButton</tabstop>
  <tabstop>lineSpinBox</tabstop>
  <tabstop>lineHighlightColorButton</tabstop>
  <tabstop>lineHighlightSpinBox</tabstop>
  <tabstop>defaultColorsButton</tabstop>
  <tabstop>bettiDotSpinBox</tabstop>
  <tabstop>persistenceDotSpinBox</tabstop>
  <tabstop>xaxisText</tabstop>
  <tabstop>yaxisText</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>okButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
